zuul.d/tempest_multinode.yaml

```diff
@@ -72,3 +72,31 @@
       - ^scenarios/centos-9/ci.yml
       - ^scenarios/centos-9/ceph_backends.yml
       - ^zuul.d/tempest_multinode.yaml
+
```

Comment:
We define periodic jobs here: https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/edpm_periodic.yaml Can we move this job definition in the `uul.d/edpm_periodic.yaml ` file?